presto-mixin/dashboards/presto-logs-overview.libsonnet

```diff
@@ -0,0 +1,32 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local logsDashboard = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+{
+  grafanaDashboards+::
+    if $._config.enableLokiLogs then {
+      local prestoLogs =
+        logsDashboard.new(
+          'Presto logs overview',
+          datasourceName='loki_datasource',
+          datasourceRegex='',
+          filterSelector=$._config.filterSelector,
+          labels=['job', 'presto_cluster', 'instance', 'level'],
+          formatParser=null,
+          showLogsVolume=true
+        )
+        {
+          panels+:
+            {
+              logs+:
+                // presto logs already have timestamp
+                g.panel.logs.options.withShowTime(false),
+            },
+          dashboards+:
+            {
+              logs+: g.dashboard.withLinksMixin($.grafanaDashboards['presto-overview.json'].links)
```

Comment:
Links aren't showing up right now on the logs dashboard on the cloud.

Reply:
Yea, will need to re import this dashboard so the uid changes from random string to name of dashboard. Code is correct, just need to update shared instance dashboard.